src/Servers/Kestrel/test/Interop.FunctionalTests/Http3/Http3RequestTests.cs

```diff
@@ -2021,7 +2021,8 @@ public async Task GET_GracefulServerShutdown_AbortRequestsAfterHostTimeout(HttpP
                 listenOptions.Protocols = protocol;
                 listenOptions.UseHttps(TestResources.GetTestCertificate());
             });
-        });
+        },
+        shutdownTimeout: TimeSpan.FromSeconds(2));
```

Comment:
2 seconds is too short for CI machines. That's why we change the timeout from `DefaultTimeout()` to 30 seconds when running in release mode (CI mode).

Reply:
I updated it to half the default timeout value. That means 2.5 seconds locally, 15 seconds in CI.